exercises/evaluation/ex_rnw/sol_train_test_split.Rnw

```diff
@@ -0,0 +1,154 @@
+\begin{enumerate}[a)]
+  \item Get the data, define a task and corresponding train-test split, and
+  predict with trained model:
+  <<echo=TRUE, message=FALSE>>=
+
+  # get data
+  library(mlbench)
+  data(BostonHousing)
+  data_pollution <- data.frame(dis = BostonHousing$dis, nox = BostonHousing$nox)
+  data_pollution <- data_pollution[order(data_pollution$dis), ]
+  head(data_pollution)
+  
+  # define task and train-test split
+  library(mlr3)
+  task <- mlr3::TaskRegr$new("pollution", backend = data_pollution, target = "nox")
+  train_set = 1:10
+  test_set = setdiff(seq_len(task$nrow), train_set)
+  
+  # train linear learner
+  library(mlr3learners)
+  learner <- mlr3::lrn("regr.lm")
+  learner$train(task, row_ids = train_set)
+  
+  # predict on test data
+  predictions <- learner$predict(task, row_ids = test_set)
+  predictions$score(mlr3::msr("regr.mse"))
+  @
+  
+  \item We have chosen the first ten observations from a data set that is 
+  ordered by feature value, which is obviously not a good idea and leads to a 
```

Comment:
"..which is not a good idea because this is not a random sample and covers only a very small area of the feature space, and leads.."